TUM Campus App/MenuTableViewCell.swift

```diff
@@ -12,10 +12,37 @@ class MenuTableViewCell: CardTableViewCell {
 
     override func setElement(_ element: DataElement) {
         if let menu = element as? CafeteriaMenu {
-            dishLabel.text = menu.name
-            priceLabel.text = ""
+            
+            //Dish Label
+            dishLabel.text = menu.details.nameWithEmojiWithoutAnnotations
+            
+            //Price Label
+            if let menuPrice = menu.price?.st {
```

Comment:
Ohh... Ok. That won't be pretty. Hard code it for now to student. We'll then check where and when we can figure that out. Guests are everyone that doesn't have an LRZ-id. Easy. But employee's will be harder to distinguish. I think we might get it with the personal info. And store it in a singleton